packages/functions/src/public-types.ts

```diff
@@ -54,6 +74,24 @@ export interface HttpsCallableOptions {
   limitedUseAppCheckTokens?: boolean;
 }
 
+/**
+ * An interface for metadata about how stream call should be executed.
+ * @public
+ */
+export interface HttpsCallableStreamOptions {
+  /**
+   * An AbortSignal that can be used to cancel the streaming response. When the signal is aborted,
+   * both the underlying connection and stream will be terminated.
```

Comment:
You are right that it would depend on the server imlementation.

We do try to provide the right primitives for developers to detect client dropping off, but who knows whether the developer would properly handle client connection drops.

See related server SDK PR: https://github.com/firebase/firebase-functions/pull/1635.